src/teamChangeMod/teamChangeMod.java

```diff
@@ -1,210 +1,181 @@
-                                if(p.team() == newTeam.team){
-                                    newTeamPlayers[0]++;
-                                };
-                            });
-                            if(newTeamPlayers[0] / Groups.player.size() * state.teams.active.size <= teamChangeTolerance){
+            try {
+                if (args.length == 1) {
+                    Team newTeam = findTeam(args[0]);
+                    if (newTeam != null && newTeam.active()) {
+                        if (state.rules.pvp && newTeam != Team.derelict && newTeam != adminTeam && !(state.rules.waves && newTeam == state.rules.waveTeam)) {
+                            float newTeamPlayers = Groups.player.count(p -> p.team() == newTeam);
+
+                            if (newTeamPlayers / Groups.player.size() * state.teams.active.size <= teamChangeTolerance) {
                                 player.clearUnit();
-                                player.team(newTeam.team);
-                                Call.sendMessage("Player " + player.name + " [white]changed team to [#" + newTeam.team.color + "]" + newTeam.team.name + "[].");
-                            }else{
-                                player.sendMessage("That team has too much players.");
-                            };
-                        }else{
-                            player.sendMessage("You can't switch to that team.");
-                        };
-                    }else{
-                        player.sendMessage("Couldn't find that team.");
-                    };
-                }else{
-                    String[] teams = new String[]{""};
-                    state.teams.active.each(t -> {
-                        teams[0] += "[#" + t.team.color + "]" + t.team.name + ", ";
-                    });
-                    player.sendMessage(teams[0]);
-                };
-            }catch(Exception badArguments){
-                player.sendMessage("Something went wrong.");
-            };
+                                player.team(newTeam);
+                                Call.sendMessage("[accent]Player " + player.coloredName() + "[accent] changed team to " + colorizedTeam(newTeam) + "[].");
+                            } else {
+                                player.sendMessage("[scarlet]That team has too much players.");
+                            }
+                        } else {
+                            player.sendMessage("[scarlet]You can't switch to that team.");
+                        }
+                    } else {
+                        player.sendMessage("[scarlet]Couldn't find that team.");
+                    }
+                } else {
+                    StringBuilder teams = new StringBuilder();
+                    Structs.each(t -> teams.append("\n[gold] - [white]").append(colorizedTeam(t)), Team.baseTeams);
+                    player.sendMessage("[scarlet]Team not found!\n[accent]Available teams:[white]" + teams);

```

Comment:
It's not supposed to say "Team not found", this is intended behavior when no arguments are provided
Also, do not make it list `baseTeams`, it's supposed to list `state.teams.active`
Same goes for the one in `/forceteam`, could be a good idea for it to list baseTeams and tell the player that they can also use any number from 6 to 255